lms/templates/courseware/course_about.html

```diff
@@ -157,6 +157,11 @@
         ## so that they can register and become a real user that can enroll.
         % elif not is_shib_course and not can_enroll:
           <span class="register disabled">${_("Enrollment is Closed")}</span>
+        %elif can_add_course_to_cart and no_id_professional_mode:
+             <a href="#" class="register">
+            ${_("Register for {course_name} ({price})").format(course_name=course.display_number_with_default,
+                 price=course_price) | h}
+          </a>
```

Comment:
This change isn't necessary: on EdX.org, we don't use the "add to cart" interface.  We also don't use this "course about" page -- instead, users enter the flow from the Drupal-based marketing site.